### PR TITLE
Remove findings matching from scancode

### PR DESCRIPTION
**Pull request by fviernau**

### CODE REVIEW

File: scanner/src/main/kotlin/scanners/Askalono.kt [hunk 1]
```diff
@@ -154,7 +154,7 @@ class Askalono(name: String, config: ScannerConfiguration) : LocalScanner(name,
         return yamlMapper.createArrayNode().apply { addAll(yamlNodes) }
     }
 
-    override fun generateSummary(startTime: Instant, endTime: Instant, scanPath: File, result: JsonNode): ScanSummary {
+    private fun generateSummary(startTime: Instant, endTime: Instant, scanPath: File, result: JsonNode): ScanSummary {
```
Comment (sschuberth) [inline scanner/src/main/kotlin/scanners/Askalono.kt]: Commit message: Change "only called class internally by each scanner implementations" to "only called internally by each scanner implementation".

File: scanner/src/test/kotlin/ScanCodeTest.kt [hunk 2]
```diff
@@ -19,12 +19,13 @@
 
 package com.here.ort.scanner
 
-import com.here.ort.model.CopyrightFindings
-import com.here.ort.model.LicenseFindings
+import com.here.ort.model.CopyrightFinding
```
Comment (sschuberth) [inline scanner/src/test/kotlin/ScanCodeTest.kt]: Commit message: "there need to be tests added FindingsMatcherTest" (I guess a "to" is missing?)

File: scanner/src/test/kotlin/ScanCodeTest.kt [hunk 3]
```diff
@@ -139,282 +140,573 @@ class ScanCodeTest : WordSpec({
...
-                            TextLocation("test/profile.js", 2, 2),
-                            TextLocation("test/regression-tests.js", 2, 2),
-                            TextLocation("test/unit-tests.js", 2, 2),
-                            TextLocation("test/utils/create-testcases.js", 2, 2),
-                            TextLocation("test/utils/error-to-object.js", 2, 2),
-                            TextLocation("test/utils/evaluate-testcase.js", 2, 2),
-                            TextLocation("tools/generate-fixtures.js", 2, 2)
-                        )
+    "generateSummary()" should {
+        // TODO: minimize this test case
```
Comment (sschuberth) [inline scanner/src/test/kotlin/ScanCodeTest.kt]: Please use properly formatted sentences even for TODOs: "Minimize this test case."

File: scanner/src/main/kotlin/scanners/Askalono.kt (hunk 1 above)
Comment (fviernau) [reply scanner/src/main/kotlin/scanners/Askalono.kt]: done.

File: scanner/src/test/kotlin/ScanCodeTest.kt (hunk 2 above)
Comment (fviernau) [reply scanner/src/test/kotlin/ScanCodeTest.kt]: done.

File: scanner/src/test/kotlin/ScanCodeTest.kt (hunk 3 above)
Comment (fviernau) [reply scanner/src/test/kotlin/ScanCodeTest.kt]: done.

File: scanner/src/main/kotlin/scanners/BoyterLc.kt [hunk 4]
```diff
@@ -153,7 +153,7 @@ class BoyterLc(name: String, config: ScannerConfiguration) : LocalScanner(name,
             EMPTY_JSON_NODE
         }
 
-    override fun generateSummary(startTime: Instant, endTime: Instant, scanPath: File, result: JsonNode): ScanSummary {
+    internal fun generateSummary(startTime: Instant, endTime: Instant, scanPath: File, result: JsonNode): ScanSummary {
```
Comment (mnonnenmacher) [inline scanner/src/main/kotlin/scanners/BoyterLc.kt]: Why not private?

Comment (fviernau) [reply scanner/src/main/kotlin/scanners/BoyterLc.kt]: done.

**Pull request merged**